src/ui/public/chrome/directives/app_switcher/app_switcher.html

```diff
@@ -9,8 +9,8 @@
       ng-href="{{ link.active ? link.url : (link.lastSubUrl || link.url) }}"
       data-test-subj="appLink">
 
-      <div ng-if="link.icon" ng-style="{ 'background-image': 'url(../' + link.icon + ')' }" class="app-icon"></div>
-      <div ng-if="!link.icon" class="app-icon app-icon-missing">{{ link.title[0] }}</div>
+      <div ng-if="link.icon" ng-style="{ 'mask-image': 'url(../' + link.icon + ')' }" class="app-icon"></div>
```

Comment:
Never really heard of mask-image, after some research, it seems it's not too well supported. Weren't we going to use just an img tag for this?